### PR TITLE
L3-1468 Add link block component

### PR DESCRIPTION
**Pull request by scottdickerson**
**Jira ticket**

[L3-1468](https://phillipsauctions.atlassian.net/browse/L3-1468)

**Screenshots**
image

**Summary**

Adds a link block component to be used in the CTA section of the about page

**Change List (describe the changes made to the files)**

- change(package.json): remove unused libraries and change react to peer and dev dependency
- change(Link): support *.phillips.com full links
- change(*.scss): use `@import` for now rather than `@use` to stop conflicts
- add(LinkBlock): Add new link block component with description
- change(reset.scss): reset the ul and li and a globally
- change(typography.scss): import Montserrat globally

**Acceptance Test (how to verify the PR)**

- Open the Link Block story and confirm the styles match the Figma for the various breakpoints

**Regression Test**

- Verify the Link story still looks correct

**Evidence of testing**

- Post logs, screenshots, etc

**Things to look for during review**

- [ ] Make sure all references to `phillips` class prefix is using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] All strings should be translatable.
- [ ] Unit test should be written and should have a coverage of 90% or higher in all areas.
- [ ] Changes or new components should either write new or update existing documentation.
- [ ] PR should link and close out an existing issue
- [ ] Make sure all commits messages follow convention and are appropriate for the changes

[L3-1468]: https://phillipsauctions.atlassian.net/browse/L3-1468?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Link/Link.tsx [hunk 1]
```diff
@@ -48,7 +48,9 @@ export interface LinkProps extends HTMLAttributes<HTMLAnchorElement> {
 const Link = ({ children, id, className, component: Component, variant, href, ...props }: LinkProps) => {
   const classNames = classnames(`${px}-link`, getLinkVariantClassName(variant ?? LinkVariants.standalone), className);
   const dataTestId = id ? `link-${id}` : `link`;
-  const isExternal = href.match(/https?/);
+  const isExternal = href.match(
+    /(http[s]?:\/\/)(?!.*phillips\.com)([a-zA-Z0-9\-.]+)(:[0-9]{1,4})?([a-zA-Z0-9/\-._~:?#[\]@!$&'()*+,;=]*)/g,
```
Comment (constantinehuzenko) [inline src/components/Link/Link.tsx:52]: can we add test cases  (as comments) what should we find using this RegEx? Would be great to have couple strings with valid cases and couple invalid

File: src/scss/_reset.scss [hunk 2]
```diff
@@ -39,6 +39,20 @@ h6 {
   overflow-wrap: break-word;
 }
 
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+ul {
```
Comment (davidicus) [inline src/scss/_reset.scss:47]: I just learned this myself. Something to keep in mind when we are authoring list https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#accessibility_concerns.

Basically we have to add `<ul role="list">` for Safari to recognize this in the accessibility tree as a list.

File: src/components/LinkBlock/LinkBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import classnames from 'classnames';
+import { px } from '../../utils';
+import Link, { LinkProps, LinkVariants } from '../Link/Link';
+
+export interface LinkBlockProps extends React.HTMLAttributes<HTMLDivElement> {
+  /** Props for the Link component */
+  linkProps: Omit<LinkProps, 'variant'>;
+  /** Renders description under link */
+  description: string;
+
```
Comment (davidicus) [inline src/components/LinkBlock/LinkBlock.tsx:10]: should we add a comment here for docs in the story?

File: src/scss/_typography.scss [hunk 4]
```diff
@@ -1,5 +1,8 @@
 @import './vars';
 
+// Design wants this replaced in the future with a loal version of Montserrat
```
Comment (davidicus) [inline src/scss/_typography.scss:3]: @typo
```suggestion
// Design wants this replaced in the future with a local version of Montserrat
```

**Review by davidicus — APPROVED** [review]
small typo

**Review by constantinehuzenko — APPROVED** [review]

**Pull request merged**